homeassistant/components/fritzbox/sensor.py

```diff
@@ -121,7 +121,7 @@ class FritzBoxPowerSensor(FritzBoxEntity, SensorEntity):
     @property
     def state(self) -> float | None:
         """Return the state of the sensor."""
-        return self.device.power / 1000  # type: ignore [no-any-return]
+        return (self.device.power or 0.0) / 1000
```

Comment:
```suggestion
        if power:= self.device power:
           return power / 1000
        return 0
```

Reply:
what is wrong with that oneliner?

Reply:
Indeed it works, but I prefer a clearer syntax. Just my opinion ;-)

Reply:
mhhh ... for me the the one-liner is fine 😄 and honestly also already used below in `FritzBoxEnergySensor` 🙈

Reply:
It works perfectly, I agree, however, I do think the suggestion by Simone is more readable.

Reply:
ok ... you  both convinced me 🙂